### ISSUE TITLE
Add ability to set unsafeArea color

### ISSUE THREAD

**Issue by teotsakalos** [thesmythgroup/easy_image_viewer#62]
When `useSafeArea: true` is set, the unsafe are has different colors than the backgroundColor that was set.
Take the following code as example:
```
showImageViewer(
          context,
          image,
          useSafeArea: true,
          immersive: true,
          swipeDismissible: true,
          doubleTapZoomable: true,
        );
```
which results in this view (
top and bottom unsafe areas have different colors):
<img src="https://github.com/thesmythgroup/easy_image_viewer/assets/63792595/715f0b45-e56a-4217-8933-7edeb495f692" height="200"  />

And you might say 'just set `useSafeArea: false`, but if this is done then the result is this:
<img src="https://github.com/thesmythgroup/easy_image_viewer/assets/63792595/4c8bd641-e69b-4501-851a-4b361d3b799f" height="200"  />

which is just as bad, because the 'x' (close) button is too far up and also doesn't align well with phones that have rounded corners since it's right at the edge.

I think the ideal outcome would be to have the safe area enabled, but the unsafe area should also be the same color as the background - or even better to be able to set the color manually.
Also the ability to completely remove the 'x' (close) button could be helpful - it can now turn transparent but it's still tappable.

**Comment by QuanchaoSong** [comment]
Same.
By setting useSafeArea to false,  then my phone looks like:

<img width="406" alt="Screenshot 2024-06-07 at 10 03 11 AM" src="https://github.com/thesmythgroup/easy_image_viewer/assets/47345588/b4226421-c5e8-4692-93c1-7f034d29b3bd">

With finger you can't close this page due to the "close" button is too out-of-the-way.

**Comment by jfahrenkrug** [comment]
@teotsakalos That's so much for such a well-written bug report, including pictures. I've been quite busy but will try to get around to this soon. It should be a pretty easy fix.

**Comment by jfahrenkrug** [comment]
Fixed in 1.5.1
